### PR TITLE
feat: Send missing analytics data from Flank to Mixpanel

### LINKED ISSUE #2030: Send missing analytics data from Flank to Mixpanel
In order to gather enough data for business metrics (#2013), we need to send additional informations from Flank.

Missing data:
- total cost
- cost by device type
- test duration
- flank version
- bundle id / package name

### PR DESCRIPTION
**Pull request by adamfilipow92**
Fixes #2030 

Mixpanel contains the following events:

- configuration
- bundle id / package id
- total cost
- device type
- flank version

## Checklist

- [X] Documented

### CODE REVIEW

File: docs/flank_mixpanel_metrics.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Mixpanel events in Flank
+
+Flank currently tracking the following events:
```
Comment (Sloox) [inline docs/flank_mixpanel_metrics.md:3]:
```suggestion
Flank is currently tracking the following events:
```

Comment (Sloox) [inline docs/flank_mixpanel_metrics.md:3]:
```suggestion
Flank is currently tracking the following events:
```

File: docs/flank_mixpanel_metrics.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- cost per device type
+- test duration
+- flank version
+
+Every event contains a session id and project id. By ```project id``` you can find all events from every execution with specific ```project id```. By ```session id``` you can find events from specific
+Flank execution.
+
+## Configuration
+
+This event contains information about configuration executed in Flank. This event allows us
```
Comment (Sloox) [inline docs/flank_mixpanel_metrics.md:17]:
```suggestion
This event contains information about the configuration executed in Flank. This event allows us to
```

File: docs/flank_mixpanel_metrics.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+track information about devices and features used in Flank.
+It could be useful to check what features are most important for the community.
+
+Fields reflect configuration names.
+
+## Bundle and package id [app_id]
+
+This event contains information about bundle id for ios and package id for android project. Additionally,
```
Comment (Sloox) [inline docs/flank_mixpanel_metrics.md:25]:
```suggestion
This event contains information about the bundle id for ios and the package id for android project. Additionally,
```

File: docs/flank_mixpanel_metrics.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+contains platform type (android, ios). This event allows implementing ```Who uses Flank``` report with additional breakdown by ```device type```.
```
Comment (Sloox) [inline docs/flank_mixpanel_metrics.md:26]:
```suggestion
contains platform type (android, ios). This event allows us to implement a ```Who uses Flank``` report with additional breakdown by ```device type```.
```

File: docs/flank_mixpanel_metrics.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Fields:
+
+- ```app_id```
+- ```device_type```
+
+## Total cost and cost per device type [devices_cost]
+
+By this event's we can implement the report ```How many millions per month in spend is Flank responsible for on Firebase Test Lab? ```
```
Comment (Sloox) [inline docs/flank_mixpanel_metrics.md:35]:
```suggestion
With these event's we can realize the report ```How many millions per month in spend is Flank responsible for on Firebase Test Lab? ```
```

File: docs/flank_mixpanel_metrics.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+Fields:
+
+- ```virtual_cost```
+- ```physical_cost```
+- ```total_cost```
+
+## Test duration [total_test_time]
+
+By this event's we can check how long tests took.
```
Comment (Sloox) [inline docs/flank_mixpanel_metrics.md:45]:
```suggestion
With these event's we can check how long tests took.
```

File: docs/flank_mixpanel_metrics.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+Fields:
+
+- ```test_duration```
+
+## Flank Version [flank_version]
+
+By this event's we can check how frequently users upgrade the Flank version.
```
Comment (Sloox) [inline docs/flank_mixpanel_metrics.md:53]:
```suggestion
By these event's we can check how frequently users upgrade the Flank version.
```

File: docs/flank_mixpanel_metrics.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+Flank currently tracking the following events:
+
+- configuration
+- bundle id and package name with information about the device type
+- total cost
+- cost per device type
+- test duration
+- flank version
+
+Every event contains a session id and project id. By ```project id``` you can find all events from every execution with specific ```project id```. By ```session id``` you can find events from specific
```
Comment (Sloox) [inline docs/flank_mixpanel_metrics.md:12]: The end of the sentence appears to be not finished.
EG:
specific...?

**Review by Sloox — CHANGES_REQUESTED** [review]
Some language changes :)

File: docs/flank_mixpanel_metrics.md (hunk 8 above)
Comment (adamfilipow92) [reply docs/flank_mixpanel_metrics.md]: Fixed, I removed the redundant new line.

Comment (Sloox) [reply docs/flank_mixpanel_metrics.md]: 👍

**Review by piotradamczyk5 — APPROVED** [review]

**Review by Sloox — APPROVED** [review]
👍

**Pull request merged**